launch/dexory_tags.launch.py

```diff
@@ -0,0 +1,31 @@
+from os import getenv
+from launch import LaunchDescription
+from launch.actions import (DeclareLaunchArgument, GroupAction,
+                            IncludeLaunchDescription, SetEnvironmentVariable)
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch_ros.actions import Node
+from launch.substitutions import LaunchConfiguration, PythonExpression
+from ament_index_python.packages import get_package_share_directory
+from launch.launch_description_sources import AnyLaunchDescriptionSource
+ARGUMENTS = []
+
+def generate_launch_description():
+    use_sim_time = LaunchConfiguration(
+        'use_sim_time', default=(getenv('SIMULATION') == 'true'))
+
+    # april_tag_detector
+    detector_launch_file = "/launch/elp_36h11.launch.yml"
+    if use_sim_time:
+        detector_launch_file = "/launch/gazebo_elp_36h11.launch.yml"
```

Comment:
Note here: I'm doing this because in simulation, the detector pipeline doesn't need to launch the `v4l` camera driver, but on the real robot we do. So we have a sim-specific bringup here that omits that part.

Reply:
Very nice to take care of the simulation part! But there is a small issue here: I think we cant do logical expression with `LaunchConfiguration` argument, probably need to use the `condition` parameter like the rest of our launch file.